setup_starport.sh

```diff
@@ -18,6 +11,12 @@ vault write secret/annex/id_rsa value=@$KEY_PATH/id_rsa
 vault write secret/annex/id_rsa.pub value=@$KEY_PATH/id_rsa.pub
 
 cat $KEY_PATH/id_rsa.pub >> ~/.ssh/authorized_keys
-rm -r $KEY_PATH
 
-echo "Setup of Starport complete. You can now set up production"
\ No newline at end of file
+function cleanup {
+    set +e
+    rm -r $KEY_PATH
+}
+trap cleanup EXIT
```

Comment:
I think we want to setup this trap just after the `mkdir` line - at the end of the script is does nothing much.

Reply:
oops, good spot. moved.